### DISCUSSION TITLE
Windows support

### DISCUSSION THREAD

**Discussion by joshmouch** [louislam/dockge#544] [Feature Request]
With AI craze, it's natural to want to find ways to share a cheap GPU in a virtualized environment.  Best I can tell, that's a Windows machine with WSL installed, and two docker daemons (one in windows for windows containers and one in WSL for Linux containers). 
I'd love that Windows support!  
Basically, in windows, you set up two contexts: one to point to the windows docker daemon (either pipe or TCP), and the second to point to the WSL docker daemon (I think this has to be TCP?  Or maybe can be piped with a UNC path....)

### ✔️ Solution

Create a windows image.

### ✔️ Alternatives
Portainer.io?
Docker Desktop?

**Comment by RyzeNGrind** [comment]
+1
